lib/src/models/subreddit_moderation.dart

```diff
@@ -116,6 +116,20 @@ enum ModeratorActionType {
   wikiUnbanned,
 }
 
+enum SpamSensitivity { all, low, high }
+enum WikiAccessMode { anyone, disabled, modonly }
+enum PostContentType { any, link, self }
+enum SuggestedCommentSort {
+  confidence,
+  controversial,
+  live,
+  newer,
```

Comment:
the actual value is `new` instead of `newer` however keywords arent allowed in `enums`

Reply:
done!!

Reply:
Awesome, thanks!